fastlane/Fastfile

```diff
@@ -25,7 +25,7 @@ platform :ios do
       clean: true,
       output_directory: './artifacts/unit-tests',
       scheme: 'MiniApp_Tests',
-      device: 'iPhone X',
+      device: 'iPhone 8',
```

Comment:
Any reason for this change?

Reply:
iPhone X is not available since Xcode 11, iPhone 8 is more universal (and the ci was picking it as it couldn't find iPhoneX simulator)

Reply:
Ok. Makes sense, just wanna know.

Reply:
Ok interesting. I changed this setting back to iPhoneX and the tests started to work. It looks like Travis is creating iPhoneX simulators up to iOS 12.4 so it was always picking this one before. With the iPhone 8, Travis used iOS 13.2 simulator and that seems to cause the issue (upgrading to Xcode 11.7 doesn't help). Anyway I'm leaving the old setting and letting you know, maybe it's some iOS 13.2 specific bug...